public/video-ui/src/components/VideoDetails/VideoDetails.js

```diff
@@ -27,6 +27,13 @@ export default class VideoDetails extends React.Component {
 
               <dt className="details-list__title">Version</dt>
               <dd className="details-list__field">{this.props.video.data.activeVersion}</dd>
+
+              <dt className="details-list__title">CAPI link</dt>
+              <dd className="details-list__field">
+                <a href={'https://preview.content.code.dev-guardianapis.com/atom/media/' + this.props.video.id + '?api-key=test'}>
```

Comment:
@jennysivapalan Noted, I need to add a proper config anyway, I'll do this in a separate PR